crates/viewer/re_viewer_context/src/viewer_context.rs

```diff
@@ -131,11 +135,14 @@ impl ViewerContext<'_> {
         self.rec_cfg.time_ctrl.read().current_query()
     }
 
-    /// Set hover/select/focus for a given selection based on an egui response.
-    pub fn select_hovered_on_click(
+    /// Consistently handle the selection, hover, drag start interactions for a given set of items.
+    ///
+    /// The drag start interaction is optional and controlled by the `draggable` parameter.
+    pub fn handle_select_hover_drag_interactions(
         &self,
         response: &egui::Response,
         selection: impl Into<ItemCollection>,
+        draggable: bool,
```

Comment:
What are the design considerations for when to set this to `true` and when to set this to `false`?

Could the value of this be inferred from the items instead?

Reply:
> Could the value of this be inferred from the items instead?

No. See my other comment.